docs/modules/ROOT/images/action_customization/timeout-logs/setup_timeout_logs.py

```
@@ -0,0 +1,42 @@
+
+def _wait_for_body_attr(driver, attr, timeout=15):
+    WebDriverWait(driver, timeout).until(
+        lambda d: bool(d.find_element(By.TAG_NAME, "body").get_attribute(attr))
+    )
+
+
+def _wait_for_timed_out_log(driver, timeout=20):
+    def has_timed_out_row(d):
+        try:
+            status = d.find_element(By.CSS_SELECTOR, ".logs-table .status-timeout").text
+        except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)